pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -102,30 +102,92 @@ let
       wrapt
   ]);
 
-  bazel-build = buildBazelPackage {
+  rules_cc_patched = stdenv.mkDerivation {
```

Comment:
Given all the patches are darwin-specific, I'd have `darwin` somewhere in the name, e.g. `rules_cc_darwin` or smth? (same for `llvm-raw_patched` below)

Reply:
Done